platform/mk3/HOW_TO_FLASH.txt

```diff
@@ -0,0 +1,34 @@
+================================
```

Comment:
Just a thought... if you write this in GitHub Markdown syntax then you will have a webpage you can point people to directly that will be rendered nicely on GitHub.  YMMV.